src/main/java/be/seeseemelk/mockbukkit/profile/PlayerTexturesMock.java

```diff
@@ -0,0 +1,111 @@
+package be.seeseemelk.mockbukkit.profile;
```

Comment:
Could you make a simple unit test for this class?

Hmmm, at least for the equals method (if you are willing to implement that). Otherwise it's mostly setters and getters, which I myself believe does not need to be unit tested (but feel free to)

Reply:
Yes they should be unit tested.